src/admin/urls.py

```diff
@@ -8,3 +9,6 @@
 urlpatterns = [
     path("self_check/", views.SelfCheckView.as_view(), name="self-check"),
 ]
+
+if settings.EMAIL_BACKEND == "anymail.backends.test.EmailBackend":
+    urlpatterns += [path("list/", views.mail_list, name="mail-list"), ]
```

Comment:
Is this view necessary? The console should be more than adequate.

Reply:
We have another thread to discuss this on